### ISSUE TITLE
Detail pages of restaurants 3 & 4 not styled properly

### ISSUE THREAD

**Issue by TinyLittleMaggie** [TinyLittleMaggie/Restaurant-Reviews#2]
It seems like the styling applied to all other restaurants isn't applied to restaurants 3 & 4.

**Comment by TinyLittleMaggie** [comment]
I also noticed that the .html files loaded for these two pages are not the latest version of the files. (e.g. I updated the link to the project CSS in the **restaurant.html**'s <head>, and also changed the copyright mark at the footer from 2017 to 2019. But when I test the project in the browser, click on restaurant 3 or 4, and inspect the page's elements, these changes are not reflected. )

Why is this happening? / Why are other pages loaded properly? / How did the old version of the html file get loaded when it's not even in the project folder anymore? 
<img width="1066" alt="Screenshot 2019-08-29 at 14 12 45" src="https://user-images.githubusercontent.com/23370593/63914884-1be5d480-ca67-11e9-9b92-0070e53081ba.png">
